lhotse/cut/set.py

```diff
@@ -2374,6 +2387,11 @@ def save_audios(
             )
             executor = None
 
+        logging.info(
```

Comment:
We generally don't emit logs in lhotse methods (for better or worse), can we remove it here?

Reply:
Ok, actually, by default the `logging.info(.)` level does not print anything. 
The first "active" verbosity level by default is : `logging.warn(.)`.

So, technically, it does not print anything, unless a debug mode is started by lhotse user...
Piotr, do you still want to remove it ?

Reply:
If lhotse is used in icefall and if icefall uses a logging level >= info, I think this message would still be printed out.

Reply:
Yes, that is also true.